Misc/NEWS.d/next/Library/2020-01-26-20-54-04.bpo-38599.0PObOD.rst

```diff
@@ -0,0 +1,2 @@
+Add Deprecation Warning message when an Queue and Locks objects are created
+and the loop is not running.
```

Comment:
I'd recommend the following changes to the `Misc/NEWS` entry:

1) Mention other classes that were affected, in addition to Queue and Lock.
2) Specify the module of the class; it's not not immediately clear if this is talking about the regular Queues and Locks, the asyncio version, threading version, or the multiprocessing version.
3) A few grammatical fixes and improvements

```
The creation of :class:`asyncio.Queue`, :class:`asyncio.Lock`,
:class:`asyncio.Event`, :class:`asyncio.Condition`, and
:class:`asyncio.Semaphore` objects outside of a running event loop has been
deprecated.
```

(I fitted the above lines to be within the 80 char limit)

Reply:
Ready thanks for the improve